lib/trento_web/controllers/v1/database_controller.ex

```diff
@@ -138,6 +138,7 @@ defmodule TrentoWeb.V1.DatabaseController do
         required: true,
         type: %OpenApiSpex.Schema{
           type: :string,
+          enum: ["database_start", "database_stop"],
```

Comment:
Let's not do this by now.
Putting enums in schemas is a pain afterwards, as we will add new operations and it will complain that we are not backward compatible.
I would only put enums in places where we are pretty sure we are not going to add new items

Reply:
We can put the available options in the description entry if needed

Reply:
Ok! Understood. Let's do it that way then, adding the options in the description, but avoiding using the `enum` field. I'll push the change soon.